examples/connext_dds/asyncwaitset/c++03/AwsExample_subscriber.cxx

```diff
@@ -97,82 +89,86 @@ void AwsSubscriber::process_received_samples()
     // samples
-    async_waitset_.unlock_condition(dds::core::cond::StatusCondition(receiver_));
+    async_waitset_.unlock_condition(
+            dds::core::cond::StatusCondition(receiver_));
 
-    // Process sample 
-    for (dds::sub::LoanedSamples<AwsExample>::iterator sample_it = samples.begin();
+    // Process sample
+    for (dds::sub::LoanedSamples<AwsExample>::iterator sample_it =
+                 samples.begin();
          sample_it != samples.end();
          sample_it++) {
         if (sample_it->info().valid()) {
-            std::cout 
-                    << "Received sample:\n\t" 
-                    << sample_it->data() 
-                    << std::endl;
-        }        
+            std::cout << "Received sample:\n\t" << sample_it->data()
+                      << std::endl;
+        }
     }
-    // Sleep a random amount of time between 1 and 10 secs. This is 
-    // intended to cause the handling thread of the AWS to take a long 
-    // time dispatching    
+    // Sleep a random amount of time between 1 and 10 secs. This is
+    // intended to cause the handling thread of the AWS to take a long
+    // time dispatching
     rti::util::sleep(dds::core::Duration::from_secs(rand() % 10 + 1));
 }
 
 int AwsSubscriber::received_count()
 {
     return receiver_->datareader_protocol_status()
-            .received_sample_count().total();
+            .received_sample_count()
+            .total();
 }
 
 AwsSubscriber::~AwsSubscriber()
 {
-    async_waitset_.detach_condition(dds::core::cond::StatusCondition(receiver_));
+    async_waitset_.detach_condition(
+            dds::core::cond::StatusCondition(receiver_));
 }
 
 
-
 int main(int argc, char *argv[])
 {
     int domain_id = 0;
     int thread_pool_size = 4;
-    int sample_count = 0; // infinite loop
+    int sample_count = 0;  // infinite loop
     const std::string USAGE(
             "AwsSubscriber_subscriber [options]\n"
             "Options:\n"
             "\t-d, -domainId: Domain ID\n"
-            "\t-t, -threads: Number of threads used to process sample reception\n"
-            "\t-s, -samples: Total number of received samples before exiting\n");
-    
+            "\t-t, -threads: Number of threads used to process sample "
+            "reception\n"
+            "\t-s, -samples: Total number of received samples before "
+            "exiting\n");
+
     srand(time(NULL));
 
     for (int i = 0; i < argc; i++) {
-        if (strcmp(argv[i], "-d") == 0
-                || strcmp(argv[i], "-domainId") == 0) {
+        if (strcmp(argv[i], "-d") == 0 || strcmp(argv[i], "-domainId") == 0) {
             if (i == argc - 1) {
                 std::cerr << "missing domain ID parameter value " << std::endl;
                 return -1;
             } else {
                 domain_id = atoi(argv[++i]);
             }
-        } else if (strcmp(argv[i], "-t") == 0
+        } else if (
+                strcmp(argv[i], "-t") == 0
```

Comment:
Seems like a limitation on clang-format. I would have expected that `strcmp()` to be on the same line as `else if(`.